Scripts/sort-schemes.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```

Comment:
99% sure that you do not need this script

Reply:
Removing